docs/examples/button/loading.vue

```diff
@@ -1,9 +1,53 @@
 <template>
-  <el-button type="primary" :loading="true">Loading</el-button>
+  <el-button type="primary" loading>Loading</el-button>
+  <el-button type="primary" :loading-icon="Eleme" loading>Loading</el-button>
+  <el-button type="primary" loading>
```

Comment:
How to set `loading-icon` to use as a string?

Reply:
> How to set `loading-icon` to use as a string?

if icon component already register, you can use string

Reply:
ok, I see